app/utils/server/responses.utils.ts

```diff
@@ -315,9 +314,9 @@ export function image(
 export async function download(filePath: string) {
   const file = new NodeOnDiskFile(filePath, getMimeType(filePath));
```

Comment:
If you move the line up here you don't have to create the `NodeOnDiskFile` at all
```suggestion
  const file = new File([await fs.readFile(filePath)], file.name, { type: file.type }
```

Reply:
You'll just have to update `file.name` to be `getFileBaseName` (I'm _pretty_ sure it's the base name but I would double check